src/EPR.PRN.Backend.API/Repositories/Repository.cs

```diff
@@ -1,12 +1,14 @@
 ﻿namespace EPR.PRN.Backend.API.Repositories
 {
     using EPR.PRN.Backend.API.Common.DTO;
+    using EPR.PRN.Backend.API.Common.Enums;
     using EPR.PRN.Backend.API.Repositories.Interfaces;
     using EPR.PRN.Backend.Data;
     using EPR.PRN.Backend.Data.DataModels;
     using Microsoft.EntityFrameworkCore;
     using Microsoft.EntityFrameworkCore.Storage;
     using System.Collections.Generic;
+    using System.Diagnostics.CodeAnalysis;
```

Comment:
Is using System.Diagnostics.CodeAnalysis required?

Reply:
Removed